docs/getting-started-on-snowplow-open-source/quick-start-aws/index.md

```diff
@@ -88,95 +96,78 @@ terraform plan
 - The default VPC you will deploy your Pipeline into. [Help](https://docs.aws.amazon.com/vpc/latest/userguide/default-vpc.html#view-default-vpc).
-- An SSL certificate so that your collector can receive HTTPS traffic on a domain you own. You can create one using [Amazon Certificate Manager](https://aws.amazon.com/certificate-manager/). Once you have done that, add the certificate in the `ssl_certificate_arn` variable.
+- (optional) An SSL certificate so that your collector can receive HTTPS traffic on a domain you own. You can create one using [Amazon Certificate Manager](https://aws.amazon.com/certificate-manager/). Once you have done that, add the certificate in the `ssl_certificate_arn` variable.
 - How to generate a SSH Key.
     - On most systems you can generate a SSH Key with: `ssh-keygen -t rsa -b 4096`
     - This will output where you public key is stored, for example: `~/.ssh/id_rsa.pub`
     - You can get the value with `cat ~/.ssh/id_rsa.pub`
 
-As mentioned above, there are two options for pipeline's destination database. These are Postgres and Snowflake. Your chosen database needs to be specified with `pipeline_db` variable. Allowed values for that variable are `postgres` and `snowflake`. Respective `<destination>.terraform.tfvars` file should be filled in according to the chosen database. Only database specific variables are different in those two tfvars files.
+As mentioned above, there are several options for the pipeline's destination database. These are Postgres, Snowflake, Redshift and Databricks. For each destination there is a dedicated block of variables that will need to be filled and the `<destination>_enabled` variable set to `true`.
 
 ##### Postgres
 
-If you choose Postgres as destination, there is no additional step. Respective variables need to be filled according to the desired setup. Necessary resources like Postgres instance, database, table, user will be created by Pipeline Terraform module.
-
-##### Snowflake
-
-If you choose Snowflake as destination, there is one more additional step. Pipeline Terraform module doesn't create necessary Snowflake resources, unlike Postgres resources. It only deploys the Snowflake Loader. Therefore Snowflake resources need to be created before deploying the pipeline. We've created [another Terraform module](https://github.com/snowplow/quickstart-examples/tree/snowflake-loader/terraform/aws/snowflake) for this purpose. You can follow the Optional Step 2 below to learn how to use Snowflake Terraform module. Snowflake Terraform module will give outputs after it is applied. These outputs need to be passed to `snowflake.terraform.tfvars`.
+If you choose Postgres as destination, there are no additional steps. Respective variables need to be filled according to the desired setup. Necessary resources like a PostgreSQL RDS instance, database, table, user will be created by the Terraform modules.
```

Comment:
Could you list the “respective variables” or at least clarify that those are `postgres_...` ? Also, I would make it clear that `postgres_enabled` should be set to `true` here.